### DISCUSSION TITLE
Account restricted for violation of terms of service?

### DISCUSSION THREAD

**Discussion by mordt22** [codewars/codewars.com#3117] [Q&A] [answered]
Hello, I just solved a level 2 kata and my kyu went to level 7 again and in my profile says i have the account restricted for violation of terms of service. Why has that happened??

**Comment by Blind4Basics** [comment]
Your account has been identified as the one of a copycat. Suggestion: delete it, start fresh (without cheating, this time).

Cheers

**Reply by mordt22** [reply]
Hi, what do you mean a copycat? I had a code that worked for the kata but it didnt fulfill the kata's requirements, so i asked a friend for help and he told me to rewrite the code in that form. I wouldn't cheat on purpose, i'm here to learn not to gain points. Couldnt you restaurate my account?

**Reply by Blind4Basics** [reply, marked as answer]
"copycat" is exactly that: using code of other people. In your case, too many occurrences of that showed up, so your account got locked.

My suggestion stays the same: delete the current account and start fresh.
